twisted/test/test_ident.py

```diff
@@ -192,3 +211,23 @@ def testNonExistingAddress(self):
         self.assertRaises(ident.NoUser, p.lookup, ('127.0.0.1', 25),
                                                   ('1.2.3.4', 763))
 
+
+    def testLookupProcNetTcp(self):
+        """
+        L{ident.ProcServerMixin.lookup} uses the Linux TCP process table.
+        """
+        open_calls = []
+
+        def mocked_open(*args, **kwargs):
+            """
+            Mock for the open call to prevent actually opening /proc/net/tcp.
+            """
+            open_calls.append((args, kwargs))
+            return NativeStringIO(self.sampleFile)
```

Comment:
minor comment

since the sampleFile is for now only used in this test, I find it easier to read if it would have been defined in this test...

with data initilized outside of the test, I have to search across the while class to find out how this tests is set up